qiskit_metal/analyses/core/base_with_renderer.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+from abc import abstractmethod
+from qiskit_metal.designs import QDesign  # pylint: disable=unused-import
+
+from . import QAnalysis
+from ... import Dict
+
+
+class QAnalysisRenderer(QAnalysis):
+    """A subclass of `QAnalysis`, intended to standardize across all Analysis classes
+    select and name renderers.
+
+    Default Setup:
+        * name (str): Name of simulation setup. Defaults to "Setup".
+
+    Data Labels:
+        * sim_setup_name (str): Name given to the current setup.
+    """
+
+    default_setup = Dict(sim=Dict(name="Setup"))
+    """Default setup"""
+
+    # supported labels for data generated from the simulation
```

Comment:
Should this go into the docstring below?

Reply:
do you meant....the documentation about the "Setup" variable? it is in the docstring above (the docstring of the class and the docstring of the __init__ are concatenated in the docs, so it does not matter in which of the two it is, thus I followed the approach we used in qlibrary, which is to list them in the class docstring)